pyproject.toml

```diff
@@ -64,13 +64,16 @@ cdsapi = "^0.7.0"
 rioxarray = "^0.15.1"
 html2text = "^2020.1.16"
 pygithub = "^2.3.0"
-pandas = "2.2.1"
+pandas = "2.2.2"
```

Comment:
Rapidfuzz is fine, but updating pandas is always tricky. Let me create a separate PR and make sure updating pandas doesn't change anything.

Reply:
@paarriagadap since the library is used only for creating the snapshot, it's fine if you don't include them in `pyproject.toml` (or keep them commented until we bump pandas version) and just install them with `pip install oda_reader`, run the snapshot once and then reinstall with `poetry install`. That should unblock you for the time being if you're in hurry.

Reply:
Makes sense. Thanks @Marigold !